### ISSUE TITLE
[SerDes 45] Add an "update this Saved Question from YAML" button on frontend

### ISSUE THREAD

**Issue by camsaul** [metabase/metabase#24781]
Part of #24766

Depends on:

- #24780

Once #24780 is in place we should add a button somewhere to the Saved Question view in the frontend UI to update it with YAML. This button should only be visible if we have the appropriate enterprise token feature. Whether we should support uploading a local YAML file or pasting into a text field or both is an open design question.

**Comment by camsaul** [comment]
Closing as no longer relevant -- we decided to go a different route with the SerDes work as part of the 45 cycle.
